### DISCUSSION TITLE
Austinp variant on windows?

### DISCUSSION THREAD

**Discussion by KRRT7** [P403n1x87/austin#209] [General]
is the https://github.com/P403n1x87/austin/blob/master/README.md#native-frame-stack austinP variant available on windows at all?

**Comment by P403n1x87** [comment]
The austinp variant uses ptrace and libunwind on Linux to perform native stack unwinding. Windows offers an API to perform a similar operation, but I haven't looked into implementing that into Austin as a variant for Windows. If you are looking for something like Austin but with support for native stacks on Windows, perhaps you could check out [Echion](https://github.com/P403n1x87/echion/tree/feat/win-support). This branch is still work in progress, but native stack unwinding on Windows should work fine (the only thing that is not working as expected is where mode, and potentially attaching to a running Python process). Note that this requires compilation from sources at the moment
